app/presenters/concerns/add_course_to_application.rb

```diff
@@ -0,0 +1,15 @@
+module AddCourseToApplication
+  extend ActiveSupport::Concern
+  VERSION = '1.3'
```

Comment:
We have to make additional checks against `ApplicationPresenter::VERSIONS` using `check_module_version` to prevent a version mismatch. Could we have a single source of truth for what is present in each version? Do the concerns need to know their version?

Reply:
Addressed in the PR implementing versioning - https://github.com/DFE-Digital/apply-for-teacher-training/pull/6170